components/article/client/view.coffee

```diff
@@ -256,6 +256,11 @@ module.exports = class ArticleView extends Backbone.View
     $(@$el).waypoint (direction) =>
       @fadeInShare() if direction is 'down'
 
+  setupMarketoStyles: ->
+    @fadeInStyles = _.once -> @$('.mktoFieldWrap input').attr('placeholder', 'Enter your email address')
```

Comment:
I'm not sure how much work `_.once` is doing here, since when calling `@setupMarketoStyles()` it generates a new `_.once`ed function each time (kinda defeating the purpose). I'd just drop `_.once` in this case since we only call `setupMarketoStyles` once anyways. Also to note—be careful with the skinny arrow here, you probably want the fat arrow `=>`, otherwise you'll lose scope to `this` (or `@` in Coffeescript).

For note in the future, if you wanted to get `_.once` to work properly in this case, you'd want to declare the wrapped function it in initialize, e.g.

```coffeescript
  initialize: ->
    @fadeInStyles = _.once => ...
  
  setupMarketoStyles: ->
    $(@$el).waypoint (direction) => @fadeInStyles() if direction is 'down'
```